sbncode/EventGenerator/MeVPrtl/Tools/ALP/ThreeBodyIntegrator.h

```diff
@@ -0,0 +1,34 @@
+/*
+
+ThreeBodyIntegator.h
+
+Class for conducting three-body integration
+
+Joshua Berger 01/22/2023
+
+*/
+
+#include <array>
+#include <functional>
+
+class ThreeBodyIntegrator {
```

Comment:
This interface is a bit dry. Can we add some comments? What are the masses? Why m122 (which I'm guess is m_{12}^2 (a la latex) is singled out?

Reply:
Fixed.